packages/admin/src/table/filterbar/FilterBarMoreFilterButton.tsx

```diff
@@ -0,0 +1,51 @@
+import { makeStyles, Theme, Typography } from "@material-ui/core";
+import { MoreHoriz } from "@material-ui/icons";
+import * as React from "react";
+import { FormattedMessage } from "react-intl";
+
+interface StyleProps {
+    fieldBarWidth: number;
```

Comment:
I'd simply call this `minWidth`, which is _way_ more descriptive

Reply:
i removed the prop, because its not necessary any more. i talked with @jamesricky and its okay that i set a default minWIdth for the fields in the style object.